CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing
+
+Welcome potential contributor of `Astar Network`! The Astar Network project (formerly known as Plasm Network) is a collection of **Open Source Projects** maintained by the Astar Team and Stake Technologies. We want to make contributing to this project as easy and transparent as possible.
+
+This document will act as a starting point for those who want to be part of the Astar Ecosystem, which includes code contribution and community contribution.
+
+## Types of Contribution
+
+We welcome any types of contributions that can improve the project/network in any shape or form, may it be directly to the Astar repository codebase, feedback, or making community contributions. You don't have to be a developer to contribute to the network.
+
+## Using GitHub
+
+The Astar Network project uses GitHub as the main source control hosting service. Most forms of communication regarding changes to the code will be done within the issue board of the repository.
+
+### Opening an Issue
+
+Contributions within GitHub can take on the following forms:
+
+- **Bug Report**: If you find any bugs or unexpected behaviors using the Astar node, please open an issue that describes the issue and other information that the developer may need when investigating.
```

Comment:
In my former job, I really emphasized that users must provide logs and whatever information is needed for us to solve the problem.

This was necessary since we'd get reports like "I tried measuring signal power and it isn't good." 😑

Maybe it's not needed in open source community but thought I'd mention it.
I'd **bold** the end of the statement 😄